dev/bender/plugins/ckeditor5/static/extensions.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals bender, CKEDITOR */
+
+'use strict';
+
+( function() {
+	// Override bender.start to wait until the "ckeditor" module is ready.
+	var originalStart = bender.start;
+	bender.start = function() {
```

Comment:
+1 and in this case it also matches the "margin around blocks" rule that we agreed on.